tangelo/linq/translator/translate_pennylane.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2021 Good Chemistry Company.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Functions helping with quantum circuit format conversion between abstract
+format and pennylane format.
+
+In order to produce an equivalent circuit for the target backend, it is
+necessary to account for:
+- how the gate names differ between the source backend to the target backend.
+- how the order and conventions for some of the inputs to the gate operations
+    may also differ.
+"""
+
+from math import pi
+
+from tangelo.helpers import deprecated
+
+
+def get_pennylane_gates():
+    """Map gate name of the abstract format to the equivalent methods of the
+    cirq class API and supported gates: https://quantumai.google/pennylane/gates.
+    """
+    import pennylane as qml
+
+    GATE_PENNYLANE = dict()
+    GATE_PENNYLANE["H"] = qml.Hadamard
+    GATE_PENNYLANE["X"] = qml.PauliX
+    GATE_PENNYLANE["Y"] = qml.PauliY
+    GATE_PENNYLANE["Z"] = qml.PauliZ
+    GATE_PENNYLANE["CX"] = qml.MultiControlledX
+    GATE_PENNYLANE["CY"] = qml.CY
+    GATE_PENNYLANE["CZ"] = qml.CZ
+    GATE_PENNYLANE["S"] = qml.S
+    GATE_PENNYLANE["T"] = qml.T
+    GATE_PENNYLANE["RX"] = qml.RX
+    GATE_PENNYLANE["RY"] = qml.RY
+    GATE_PENNYLANE["RZ"] = qml.RZ
+    GATE_PENNYLANE["CNOT"] = qml.CNOT
+    GATE_PENNYLANE["CRZ"] = qml.CRZ
+    GATE_PENNYLANE["CRX"] = qml.CRX
+    GATE_PENNYLANE["CRY"] = qml.CRY
+    GATE_PENNYLANE["PHASE"] = qml.PhaseShift
+    GATE_PENNYLANE["CPHASE"] = qml.CPhase
+    GATE_PENNYLANE["XX"] = qml.IsingXX
+    GATE_PENNYLANE["SWAP"] = qml.SWAP
+    GATE_PENNYLANE["CSWAP"] = qml.CSWAP
+    # GATE_PENNYLANE["MEASURE"] = qml.measure  # Pennylane currently only supports measuring a qubit once
+    return GATE_PENNYLANE
+
+
+@deprecated("Please use the translate_circuit function.")
```

Comment:
Does it make sense for you to define a brand new function but tag it as deprecated ?

The reason other similar functions still exist and are deprecated is to not break existing code, yet tell users they should stop using these functions.

I think this function should be entirely removed.